proxy/installer/fetch-certificate.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python -u
```

Comment:
I'm not sure if it really make sense to enforce unbuffered STDOUT/STDERR streams with `-u`, which is not used for the bundle on calling `/usr/lib/venv-salt-minion/bin/python` with `configure-proxy.sh`. And please check the mod of the file, it looks like you've removed shebang here, but the file is still set as 0755 with the spec. And it's not so good to put it to `bin` with no exec mod.

Reply:
it should looks better now, thanks for your suggestion!

Reply:
Thanks. It's better, but there is still the difference with `-u` for non-bundled python. Does it make any sense there to force unbuffered streams? If does, `-u` should be added to the bundled pythin call also.

Reply:
correct, I forgot to check it. It seems that `-u` does not provide any further information (I also tried to force an error condition), so I remove it from `python3`

Reply:
Sorry for one more nitpick. Does the space at the end of `PYTHON_ENV` make any sense? I would also suggest to rename the variable to `PYTHON_CMD` as ENV is not really related to environment.

Reply:
Thanks for addressing all the suggestions :+1: